src/main/java/com/blackducksoftware/integration/hub/api/component/ComponentRestService.java

```diff
@@ -48,4 +50,20 @@ public ComponentRestService(final RestConnection restConnection, final Gson gson
         return allComponents;
     }
 
+    public ComponentItem getExactComponentMatch(String id, String groupId, String artifactId, String version)
```

Comment:
We'd want to hold off applying any name changes until the Hub is updated to take a different value. Currently, a format that seems to be supported is "q=id:maven|org.apache.log4j|log4j|2.0.0" so keeping it as 'id' should be fine. If it should be something different, please make the change in the Hub API and documentation and our client will follow.